app/views/projects/_header.html.erb

```diff
@@ -1,7 +1,11 @@
 <h1 class="project-title">
   <%= star_for_project(project) %>
   <%= project.name %>
-  <%= render partial: "shared/github_link", locals: { project: project } %>
+  <% if project.github? %>
+    <%= render partial: 'shared/github_link', locals: { project: project } %>
+  <% elsif project.gitlab? %>
+    <%= render partial: 'shared/gitlab_link', locals: { project: project } %>
+  <% end %>
```

Comment:
I think showing no link is appropriate for else, because I can't decide web page url to show.